backend/dataall/base/cdkproxy/cdk.context.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
what is this? Is this getting created for everyone or only those who enable the module?

Reply:
@ironspur5 this file needs to be deleted (clean-up)